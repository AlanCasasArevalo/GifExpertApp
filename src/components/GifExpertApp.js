import React, { useState } from 'react'
import PropTypes from 'prop-types'

const GifExpertApp = () => {

    return (
        <>
            <h1>GifExpertApp</h1>
            <hr/>
        </>
    )
}
//
// GifExpertApp.propTypes = {
//     value: PropTypes.number
// }

export default GifExpertApp;

