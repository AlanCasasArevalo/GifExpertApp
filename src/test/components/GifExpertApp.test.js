import {shallow} from "enzyme";
import React from "react";
import GifExpertApp from "../../components/GifExpertApp";

describe('GifExpertApp Component test', () => {

    let wrapper = shallow(
        <GifExpertApp
        />)

    beforeEach(() => {
        wrapper = shallow(
            <GifExpertApp
            />)
    })

    test('GifExpertApp should be defined', () => {
        expect(<GifExpertApp />).toBeDefined()
    })
    test('GifExpertApp should be an object', () => {
        expect(typeof <GifExpertApp />).toBe('object')
    })

    test('GifExpertApp should return a snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })
})
