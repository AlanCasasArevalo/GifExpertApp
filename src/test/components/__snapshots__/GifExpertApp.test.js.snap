// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifExpertApp Component test GifExpertApp should return a snapshot 1`] = `
<Fragment>
  <h1>
    GifExpertApp
  </h1>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Dragon"
      key="Dragon"
    />
  </ol>
</Fragment>
`;
